src/elements/content-sidebar/SidebarFileProperties.js

```diff
@@ -43,7 +43,7 @@ const SidebarFileProperties = ({
     return (
         <LoadingIndicatorWrapper isLoading={isLoading}>
             <ItemProperties
-                archivedAt={archiveDate && archiveDate * 1000}
+                archivedAt={archiveDate && archiveDate * 1}
```

Comment:
I would prefer casting to number explicitly

Reply:
Done